tests/utils_tests/test_simplelazyobject.py

```diff
@@ -14,3 +18,41 @@ def test_pickle(self):
         pickle.dumps(x, 0)
         pickle.dumps(x, 1)
         pickle.dumps(x, 2)
+
+    async def test_async_retrieval(self):
```

Comment:
Just FYI the Django coding standard asks that we don't write test docstrings with "Ensure that x does y" or "Test that foo does bar" 🙂  Usually the test method should be enough to describe the conditions but if it isn't enough some comments to elaborate are welcomed.

Also writing comments that describe what the next line does are discouraged unless it's unclear (as per the document-the-why-not-the-what guidelines). I think if the code is unclear what it's testing then some elaboration in the docstring may be helpful 👍